metaci/testresults/robot_importer.py

```diff
@@ -13,6 +15,20 @@ def import_robot_test_results(build_flow, path):
     # import is here to avoid import cycle
     from metaci.build.models import BuildFlowAsset
 
+    path = pathlib.Path(path)
+
+    # find the robot task that generated the log file so
+    # that we can access the task options when rendering
+    # the log file
+    mtime = timezone.make_aware(
+        datetime.fromtimestamp(path.stat().st_mtime), timezone.get_current_timezone()
+    )
+    robot_task = None
+    for task in build_flow.tasks.order_by("-time_end"):
```

Comment:
you could add `.filter(class_path="Robot")` to only get the robot tasks. (It should be `cumulusci.tasks.robotframework.Robot` but it's not getting set correctly here https://github.com/SFDO-Tooling/MetaCI/blob/master/metaci/build/flows.py#L18)

Reply:
I suppose you're right, but the filter doesn't really buy us much here since there will likely only ever be a handful of tasks and we're often going to find the one we're looking for very early in the loop.